### PR TITLE
Split e2e test cases

### LINKED ISSUE #932: CI: Split e2e test cases to reduce overall time consumption
Currently, it takes more than an hour to run a complete e2e test case, which is a bad experience both for the developer experience and for the development progress.

Although we can specify to run specific test cases before submitting code, in a CI environment, we want to run all cases in full to ensure the quality of our code.

I suggest that we can split and run the current test cases to reduce the overall time-consuming.

### PR DESCRIPTION
**Pull request by fgksgf**
### Type of change:

Split e2e test cases to reduce overall time consumption.

### What this PR does / why we need it:

Close #932 

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix-ingress-controller#community) first**

### CODE REVIEW

**Comment by tao12345666333** [comment]
2022-04-06 14-54-04屏幕截图

LGTM! Thanks!!! The CI time cut in half

**Review by lingsamuel — APPROVED** [review]

File: test/e2e/suite-features/global_rule.go [hunk 1]
```diff
@@ -17,14 +17,15 @@ package features
 import (
 	"time"
 
-	"github.com/apache/apisix-ingress-controller/pkg/id"
 	"github.com/onsi/ginkgo"
 	"github.com/stretchr/testify/assert"
 
+	"github.com/apache/apisix-ingress-controller/pkg/id"
+
```
Comment (tao12345666333) [inline test/e2e/suite-features/global_rule.go:24]: we can remove this blank line.

**Review by tao12345666333 — APPROVED** [review]
Thanks! 
LGTM

**Comment by tao12345666333** [comment]
Since #942 was recently merged, are you free to help resolve the conflict? @fgksgf

**Comment by fgksgf** [comment]
> Since #942 was recently merged, are you free to help resolve the conflict? @fgksgf

No problem.

**Pull request merged**